manifests/init.pp

```diff
@@ -17,6 +19,20 @@
     purge   => $config_dir_purge,
   }
 
+  if str2bool($cron_purge) {
```

Comment:
Isn't `$cron_purge` guarenteed to be a boolean at this point because of `validate_bool()`?